packages/rancher-vsphere-cpi/charts/values.yaml

```diff
@@ -14,6 +14,11 @@ cloudControllerManager:
     repository: rancher/mirrored-cloud-provider-vsphere-cpi-release-manager
     tag: v1.2.1
   nodeSelector: {}
+  rbac:
+    enabled: true
+
+# Set to "rke" or "rke2" to apply the node selector label appropriate for the cluster
+clusterType: "rke"
 
 global:
   cattle:
```

Comment:
These changes look fine if its a new chart version. But that begs the question why we are bumping this to 100.0.0 if its a local chart without an upstream version ? Why not move to 2.0.0.
cc @aiyengar2 and @jiaqiluo

Reply:
CSI doesn't have an upstream, both CSI and CPI are just a bunch of manifests upstream. As for the version, I thought we did 100.0.0 to be consistent with the others, but either way works so I'll leave that to you guys to decide.

Reply:
@PennyScissors is right that the initial purpose was consistency. We moved all charts to 100.0.0 unless specifically directed otherwise since diverting from 100.0.0 is an exception, not the norm, and keeping the chart major version tied to the Rancher release makes it easy for us to call out if a given chart is the right one to be installed in the given version of Rancher.

I think it's fine to move CPI / CSI back to 2.0.0 though if that is preferred for local charts.

Reply:
Currently, we have CPI 1.0.0 and CSI 2.1.0 in the release-v2.6 branch. So if we decide to move them back, I think it would need to be CPI 2.0.0 and CSI 3.0.0. I like 100.0.0 because is not only consistent with other charts, but also consistent with each other since they need to be installed together.